### PR TITLE
[WIP] Spice up voting

### PR DESCRIPTION
**Pull request by kirubarajan**
- Updated `README.md` with additional help
- Fix in-line documentation with a fresh pair of eyes
- Cleaning import structures (e.g. removing unnecessary imports)
- Fixing up UI of voting page

### CODE REVIEW

**Comment by kirubarajan** [comment]
Prevote page looks like this now:

Screen Shot 2019-11-23 at 7 39 08 PM

Not exactly what is mocked on Figma, but in a lot better shape with regards to UI structure to facilitate design hand-off. Notable functional change is the conversion of camel case in the UI to a more natural language feel (e.g. changing `"marketFit"` to `"Market Fit"`).

**Comment by kirubarajan** [comment]
Final Vote page looks like:

Screen Shot 2019-11-23 at 7 58 18 PM

Changes include some spacing changes, better UI structure (e.g. wrapping loose text in `<p>` tags).

**Comment by kirubarajan** [comment]
Wondering how to fix this error:

```
ERROR in /Users/arun/Developer/drf/magic-wand/client/components/Voting/VotingForm.tsx(15,34):
TS2339: Property 'title' does not exist on type 'JSONSchema6Definition'.
  Property 'title' does not exist on type 'false'.
Version: typescript 3.4.1
```

since `JSONSchema6Definition` isn't defined by us.

**Comment by VinaiRachakonda** [comment]
> Wondering how to fix this error:
> 
> ```
> ERROR in /Users/arun/Developer/drf/magic-wand/client/components/Voting/VotingForm.tsx(15,34):
> TS2339: Property 'title' does not exist on type 'JSONSchema6Definition'.
>   Property 'title' does not exist on type 'false'.
> Version: typescript 3.4.1
> ```
> 
> since `JSONSchema6Definition` isn't defined by us.

Not sure how to fix myself. You'll need to comb the reactjson-schema docs

File: client/components/Voting/VotingForm.tsx [hunk 1]
```diff
@@ -11,6 +11,10 @@ export interface VotingFormProps {
   disabled?: boolean;
 }
 
+Object.keys(voteFormSchema.properties).forEach((key) => {
```
Comment (VinaiRachakonda) [inline client/components/Voting/VotingForm.tsx]: A comment here would be nice

**Review by VinaiRachakonda — APPROVED** [review]
Merge when errors are gone

**Comment by VinaiRachakonda** [comment]
Screen Shot 2020-02-16 at 12 06 02 PM
Actually not seeing spacing between the vote and the final vote

**Review by VinaiRachakonda — APPROVED** [review]

**Pull request merged**